zktest/src/archive/test2/config.properties

```diff
@@ -3038,6 +3038,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##manually##B96-ZK-4535.zul=A,E,Toolbar,Toolbarbutton,Combobutton
 ##manually##B96-ZK-4944.zul=A,E,Embedded,page,destory,detach
 ##manually##B96-ZK-4953.zul=A,E,Embedded,Include,defer
+##manually##B96-ZK-4868.zul=A,M,signature,backgroundColor

```

Comment:
Try to get the canvas color: https://stackoverflow.com/questions/6735470/get-pixel-color-from-canvas-on-mousemove